web/crux-ui/src/validations/container.ts

```diff
@@ -319,6 +319,16 @@ const markerRule = yup
   .nullable()
   .optional()
 
+const metricsRule = yup
```

Comment:
We need an extra validation to check if the port exists in the container config, or another solution could be to change port 'text-input' to a 'select' only listing existent exposed ports.
If the port is not in the exposed ports list, service monitors will not be deployed.